src/builtins.ts

```diff
@@ -185,6 +185,8 @@ export namespace BuiltinNames {
   export const instantiate = "~lib/builtins/instantiate";
   export const idof = "~lib/builtins/idof";
 
+  export const add = "~lib/builtins/add";
```

Comment:
I guess better move it before `export const clz = "~lib/builtins/clz";` (line 143)

Reply:
Makes sense. Done